client/game/SocialExposure.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Slider from "meteor/empirica:slider";
+
+export default class SocialExposure extends React.Component {
+  renderSocialInteraction(otherPlayer) {
+    const value = otherPlayer.round.get("value");
+    return (
+      <div className="alter" key={otherPlayer._id}>
+        <img src={otherPlayer.get("avatar")} className="profile-avatar" />
+        Guess: {value}
+      </div>
+    );
+  }
+
+  render() {
+    const { game, player } = this.props;
+
+    const otherPlayers = _.filter(
+      game.players,
+      p =>
+        p._id != player._id && player.get("neighbors").includes(p.get("nodeId"))
+    );
+
+    window.player = player;
+    window.op = otherPlayers;
+
+    if (otherPlayers.length === 0) {
+      return null;
+    }
+
+    return (
+      <div className="social-exposure">
+        <p>
+          <strong>There are {otherPlayers.length} other players:</strong>
+        </p>
+        {!_.isEmpty(otherPlayers)
+          ? otherPlayers.map(p => this.renderSocialInteraction(p))
+          : ""}
```

Comment:
i'm reluctant to make this change here first, because it's an edit from the template---and thus it requires more work of the person following the demo

should be removed from the template first

Reply:
i seem to be unable to start an issue for create-empirica-app

i would be happy to submit a pull request but, i don't know stylistically which of the redundancies you want to remove

Reply:
going to bed now, will do tomorrow

Reply:
this update is pushed!